### ISSUE TITLE
Improve appearance and text of "not foi" request status

### ISSUE THREAD

**Issue by henare** [openaustralia/righttoknow#650]
When you use the built-in admin to hide a request for personal information (that I only recently discovered and mentioned in https://github.com/mysociety/alaveteli/issues/3543) the request gets set to "not foi" and the message displayed to the request if they're logged in looks like:

![selection_001](https://cloud.githubusercontent.com/assets/48945/19505203/725e46fe-960c-11e6-89a2-84de1a339b7d.png)

I don't really mind that the status is "not foi" in the backend but that's not strictly true in Australia (you can make FOI requests for personal information) and the text is quite abrupt. It would be nice to improve this so it's not as harsh and make the appearance more salient.

**Comment by EkaterinaSemenova** [comment]
Hi @henare, @benrfairless!
I suggest the following edit to this text: "The request has been hidden from the site, because it does not comply to our <a href="https://www.righttoknow.org.au/help/requesting">Policy</a> (e.g., it may request information about yourself or another private individual or be otherwise sensitive, or may be not focused). You may amend your request to comply with our <a href="https://www.righttoknow.org.au/help/requesting">Policy</a> and submit it as new”. Does it sound right to you now?

**Comment by benrfairless** [comment]
@ekaterinasemeonva thanks for your input. I've never been a big fan of the word policy, but like the word "guidelines" (that's what we use in our emails).

Don't forget that Right to Know looks after GIPA, RTI, and FOI requests as well, so we need to ensure we don't "hard code" the word FOI anywhere.

Perhaps: "This request has been hidden from the site as it goes against our guidelines. Please contact us if you have any questions."

What do you think?

**Comment by EkaterinaSemenova** [comment]
@benrfairless, I think there is no need to repeat "Please contact us if you have any questions.", since user already sees similar red banner at the very top of the page. 
![untitled-1](https://cloud.githubusercontent.com/assets/20787120/24570671/66e276ee-1675-11e7-9c74-a2a1d0f687d4.png)
I suggest the following: "This request has been hidden from the site as it goes against our guidelines. You may amend your request and submit it as new”.

**Comment by benrfairless** [comment]
@EkaterinaSemenova good pick up with the contact us part.

I don't like the idea of saying  _You may amend your request and submit it as new_ as a) you can't amend a request once it's submitted and b) usually requests we hide aren't suitable for Right to Know and won't ever be suitable for Right to Know.

How about:
> Our team have reviewed your request and hidden it because it goes against our guidelines.

I'm not great with wordings however, so maybe @equivalentideas can provide some additional help?

**Comment by EkaterinaSemenova** [comment]
@benrfairless, 

> I don't like the idea of saying You may amend your request and submit it as new as a) you can't amend a request once it's submitted and b) usually requests we hide aren't suitable for Right to Know and won't ever be suitable for Right to Know.

ok, makes sense. How about this wording: "Unfortunately, we had to hide your request from the site, since it goes against our guidelines. Whether you submit new request, please, make sure it complies with the guidelines."
If you do not like this wording, I can go ahead and submit merge request with wording you proposed.

**Comment by henare** [comment]
@EkaterinaSemenova @benrfairless I really like the process you both went through to refine this wording :star2: 

As @EkaterinaSemenova said, we already have the big red message that tells the request what's happening and what they can do next. The main problem with the message is the thing about it being "not an FOI request".

How about we keep it simple and just remove that bit? Something like:

>This request has been **hidden** from the site by an administrator.

Don't forget we also need to fix the appearance of this message when we make this change. It's currently a bit wonky and doesn't match the appearance of the other messages that usual appear here. Maybe because this doesn't have a status icon associated with it?

**Comment by EkaterinaSemenova** [comment]
Hi @henare!
#693 should fix it. Please, take a look.
